repository-info-extension/vss-extension.json

```diff
@@ -0,0 +1,67 @@
+{
+    "manifestVersion": 1,
+    "id": "repository-info-extension",
+    "version": "1.0.0",
+    "name": "Repository Information Sample",
+    "description": "Provides an example for accessing the currently selected repository from an extension.",
+    "publisher": "ms-samples",
+    "public": true,
+    "categories": [
```

Comment:
You should add "Code" as a category here too so it shows up in "repos" extensions